HostDependencies.md

```diff
@@ -131,16 +132,6 @@ The python dependencies required by the CAmkES build toolchain can be installed
 pip install --user camkes-deps
 ```
 
-### Haskell Dependencies (Duplicated)
```

Comment:
These instructions are still required for Camkes. Can you just remove the original change of adding "Duplicated" in the title?

Reply:
Deleted in latest commit